src/index.html

```diff
@@ -17,11 +18,368 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="../src/images/logo.png"
+            alt="Site logo"
+          />
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link header__link--active is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__item"
+            data-qa="nav-hover"
```

Comment:
The 'data-qa' attribute should be 'data-qa="nav-hover"' for the 4th nav link ('Laptops & Computers') as specified in the task requirements.